### PR TITLE
feat(1419): more pr info

### PR DESCRIPTION
**Pull request by DekusDenial**
more PR info the existing methods

blocked by: https://github.com/screwdriver-cd/data-schema/pull/303
related: https://github.com/screwdriver-cd/screwdriver/issues/1419

### CODE REVIEW

**Comment by DekusDenial** [comment]
test failed due to blocked-by upstream PR

File: index.js [hunk 1]
```diff
@@ -362,7 +362,10 @@ class ScmBase {
                 validate(jobList, Joi.array().items(
                     Joi.object().keys({
                         name: Joi.reach(dataSchema.models.job.base, 'name').required(),
-                        ref: Joi.string().required()
+                        ref: Joi.string().required(),
+                        username: Joi.reach(dataSchema.core.scm.pr, 'username'),
```
Comment (jithin1987) [inline index.js]: are these by default optional ? Just want to make sure that for scm's that have not implemented these fields yet, won't break on rebuild.

**Comment by tkyi** [comment]
Couple small comments
- Might want to bump major version for scm-base when we merge this (see https://github.com/screwdriver-cd/scm-base/pull/64 for semantic-release title format)
- Also need to update scm-router; something similar to this: https://github.com/screwdriver-cd/scm-router/pull/15

File: index.js (hunk 1 above)
Comment (DekusDenial) [reply index.js]: yeah, optional by default unless require explicitly

**Comment by DekusDenial** [comment]
a minor bump will do right since this is adding optional fields...
and https://github.com/screwdriver-cd/scm-router/blob/aad6818b570cd255897b78cc8b08ed3a67bfc461/package.json#L46 is good until another major bump 😉

**Pull request merged**